### PR TITLE
nit: please remove dot imports #1119

### LINKED ISSUE #1119: nit: please remove dot imports
Many (all?) restic tests use dot imports such as `import . "restic/test"`. This makes it harder than necessary to find the definition of identifiers, for example while debugging test failures for #1085.

The Go wiki gives similar advice: https://github.com/golang/go/wiki/CodeReviewComments#import-dot

Please get rid of dot imports.

### PR DESCRIPTION
**Pull request by ghost**
This pull request was created to remove all dot imports found in `cmd` and `internal` files.

Closes #1119

### CODE REVIEW

**Comment by hgfischer** [comment]
I don't get why the difference with Codecov. I've actually haven't added or removed code.

**Comment by hgfischer** [comment]
And it looks like Travis failed because of timeouts.

**Comment by prattmic** [comment]
+1 to removing dot imports. The uses here don't meet the circular dependency bar: https://github.com/golang/go/wiki/CodeReviewComments#import-dot

The code coverage complaint is normal. Travis skips integration tests that use keys to cloud storage providers in pull requests, which confuses CodeCov.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 /restic
 /.vagrant
 /doc/_build
+/.vscode
```
Comment (fd0) [inline .gitignore]: This does not belong here. If you want to ignore that dir in your checkout of the repo, add the directory to `.git/info/exclude`. We don't collect ignores for various editors here :)

**Review by fd0 — CHANGES_REQUESTED** [review]
Good work so far!

File: .gitignore (hunk 1 above)
Comment (hgfischer) [reply .gitignore]: I honestly think it is an exaggeration to be picky with `.gitignore`, but, whatever. It is your project.

**Comment by fd0** [comment]
@hgfischer The backend tests cannot be run because for security reasons Travis doesn't expose the needed credentials, don't worry about that.

**Review by fd0 — APPROVED** [review]

**Pull request merged**